src/Groups/GAPGroups.jl

```diff
@@ -1447,6 +1447,56 @@ see [`minimal_normal_subgroups`](@ref).
 """
 @gapattribute socle(G::GAPGroup) = _as_subgroup(G, GAP.Globals.Socle(GapObj(G)))
 
+"""
+    p_rump(G::GAPGroup, p::IntegerUnion)
+
+For a prime p, the p-rump of a group `G` is the subgroup `G'` of `G^p`.
```

Comment:
Missing backticks around `p`, which is an argument. Also, this definition is not correct.
```suggestion
For a prime `p`, the `p`-rump of a group `G` is the subgroup `G' G^p`.
```